auth-web/src/views/auth/staff/IncorporationSearchResultView.vue

```diff
@@ -176,8 +177,17 @@ export default class IncorporationSearchResultView extends Vue {
     return orgTypeDisplay
   }
 
-  async entityDashboardEvent () {
-    window.location.href = `${ConfigHelper.getBusinessURL()}${this.currentBusiness.businessIdentifier}`
+  async manageBusinessEvent () {
+    const businessIdentifier = this.currentBusiness.businessIdentifier
+    const filingId = this.filingID
+
+    let url = `${ConfigHelper.getBusinessURL()}${businessIdentifier}`
+
+    if (filingId) {
+      url += `?filingId=${filingId}`
```

Comment:
This should actually be `filing_id=`.

Example: `https://dev.business.bcregistry.gov.bc.ca/BC0884782?filing_id=155306`